File: app/src/test/java/com/mgaetan89/showsrage/fragment/SettingsAboutShowsRageFragmentTest.java
package com.mgaetan89.showsrage.fragment;

import com.mgaetan89.showsrage.R;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class SettingsAboutShowsRageFragmentTest {
	@Test
	public void getTitleResourceId() {
		assertThat(new SettingsAboutShowsRageFragment().getTitleResourceId()).isEqualTo(R.string.app_name);
	}

	@Test
	public void getXmlResourceFile() {
		assertThat(new SettingsAboutShowsRageFragment().getXmlResourceFile()).isEqualTo(R.xml.settings_about_showsrage);
	}
}

File: app/src/test/kotlin/com/mgaetan89/showsrage/fragment/SettingsAboutShowsRageFragmentTest.kt
package com.mgaetan89.showsrage.fragment

import com.mgaetan89.showsrage.R
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class SettingsAboutShowsRageFragmentTest {
    @Test
    fun getTitleResourceId() {
        assertThat(SettingsAboutShowsRageFragment().getTitleResourceId()).isEqualTo(R.string.app_name)
    }

    @Test
    fun getXmlResourceFile() {
        assertThat(SettingsAboutShowsRageFragment().getXmlResourceFile()).isEqualTo(R.xml.settings_about_showsrage)
    }
}
